service/conf/routes

```diff
@@ -2,11 +2,9 @@
 # This file defines all application routes (Higher priority routes first)
 OPTIONS    /*all                        @controllers.LearnerController.preflight(all)
 # ~~~~
-GET      /v1/user/courses/:uid	        @controllers.LearnerController.getEnrolledCourses(uid:String)
 GET      /v1/user/courses/list/:uid	        @controllers.LearnerController.getEnrolledCourses(uid:String)
 POST     /v1/user/courses/enroll	    @controllers.LearnerController.enrollCourse()
 POST     /v1/course/enrol	    		@controllers.LearnerController.enrollCourse()
-POST     /v1/user/content/state	        @controllers.LearnerController.getContentState()
```

Comment:
Check if portal or mobile is using /v1/user/content/state.

Reply:
onboarded  uri is /v1/content/state/read , actually inside routes /v1/user/content/state and /v1/content/state/read is pointing to same Controller method.